pkg/kv/kvserver/allocator_scorer.go

```diff
@@ -448,6 +547,8 @@ type candidate struct {
 	store          roachpb.StoreDescriptor
 	valid          bool
 	fullDisk       bool
+	highReadAmp    bool
```

Comment:
is this still needed, is it derived from the next int ?

Reply:
The candidate struct is (I think) meant to unaware of details further up (only data) - it gets populated as the ranking function iterates over stores. So it shouldn't worry about what defines "high read amp", the fn populating the struct does.

The next integer is there to break ties in compare().